docs/modules/pyfar.dsp.filter.rst

```diff
@@ -1,11 +1,13 @@
 pyfar.dsp.filter
 ================
 
-The following documents the pyfar filters. Visit
-:py:mod:`filter types <pyfar._concepts.filter_types>` for an introduction of
-the different filters and
-:py:mod:`filter classes <pyfar._concepts.filter_classes>` for more information
-on pyfar filter objects.
+The following documents the pyfar filters. Visit the
+`filter types <https://pyfar-gallery.readthedocs.io/en/latest/gallery/
+interactive/pyfar_filter_types.html>`_ for an overview of
+the different filters and the
+`filtering examples <https://pyfar-gallery.readthedocs.io/en/latest/
+gallery/interactive/pyfar_filtering.html>`_ for more information
+on using pyfar filter objects.
```

Comment:
```suggestion
The following documents the pyfar filters. Visit the
:doc:`gallery:gallery/interactive/fast_fourier_transform` for an overview of
the different filters and the :doc:`gallery:gallery/interactive/pyfar_filtering`
for more information
on using pyfar filter objects.
```
we need to add 
```
    'gallery': ('https://pyfar-gallery.readthedocs.io/en/latest/', None),
```
to the config.py in ``intersphinx_mapping`` then we can use intersphinx to generate the links